.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,26 @@
+<!--
+Please make sure you've read and understood our contributing guidelines;
+https://github.com/moby/moby/blob/master/CONTRIBUTING.md
```

Comment:
Should we change this to `https://github.com/docker/swarmkit/blob/master/CONTRIBUTING.md` instead?

Reply:
Ah good catch, I didn't know we had a `CONTRIBUTING.md` here too. Fixed.